packages/core/src/lib/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,181 @@
+
+    expect(div.classList.contains('filled')).toBeFalsy;
+    expect(div.classList.contains('empty')).toBeTruthy;
+
+    input.focus();
+    fireEvent.change(input, { target: { value: 22  } });
+
+    expect(Number(input.value)).toBe((22));
+    expect(testEvent).toHaveBeenCalledWith(input);
+});
+
+
+describe('Slider - Default', () => {
+
+    const testLabel = Default?.args?.rangeLabel;
+
+    beforeEach(() => {
+        container = document.createElement('div');
+
+        render(
+            <Default
+                onChanges={(event: React.ChangeEvent<HTMLInputElement>) =>
+                    testEvent(event.target)
+                }
+            />,
+            {
+                container: document.body.appendChild(container),
+            },
+        );
+    });
+
+    afterEach(() => {
+        cleanup();
+    });
+
+    it('renders the component', () => {
+        const input: HTMLInputElement | null = container.querySelector(
+            `[data-testid="${testInputId}"]`,
+        );
+        expect(input).not.toBeNull();
+    });
+
+    it('renders input with the rangeValue passed', () => {
+        const input: HTMLInputElement | null = container.querySelector(
+            `[data-testid="${testInputId}"]`,
+        );
+        expect(input).not.toBeNull();
+        input && expect(Number(input.value)).toBe(baseTestValue);
+        
+    });
+
+    it('renders the correct type of input', () => {
+        const input: HTMLInputElement | null = container.querySelector(
+            `[data-testid="${testInputId}"]`,
+        );
+        expect(input).not.toBeNull();
+        input && expect(input.type).toBe('range');
+    });
+
+    it('renders label text', () => {
+        const label = container.querySelector(`[data-testid="${testLabelId}"]`);
+        expect(label).not.toBeNull();
+        expect(label?.textContent).toBe(testLabel);
+    });
+
+})
+
+describe('Slider - Step', () => {
+
+    const testLabel = Default?.args?.rangeLabel;
+
+    beforeEach(() => {
+        container = document.createElement('div');
+
+        render(
+            <Step
+                onChanges={(event: React.ChangeEvent<HTMLInputElement>) =>
+                    testEvent(event.target)
+                }
+            />,
+            {
+                container: document.body.appendChild(container),
+            },
+        );
+    });
+
+    afterEach(() => {
+        cleanup();
+    });
+
+    it('renders the component', () => {
+        const input: HTMLInputElement | null = container.querySelector(
+            `[data-testid="${testInputId}"]`,
+        );
+        expect(input).not.toBeNull();
+    });
+
+    it('renders the component with step value passed', () => {
+        const input: HTMLInputElement | null = container.querySelector(
+            `[data-testid="${testInputId}"]`,
+        );
+        expect(input?.step).not.toBeNull();
+        expect(input?.step).toBe(testStepValue)
+    });
```

Comment:
This test is failing.
You can run `pnpm vitest --ui`  to run all tests and it will give you a UI which shows all tests and it's status